packages/roosterjs-editor-dom/lib/list/VList.ts

```diff
@@ -131,6 +137,15 @@ export default class VList {
         queryElements(this.rootList, 'li', moveLiToList);
 
         this.populateItems(this.rootList);
+        this.config = DEFAULT_VLIST_CONFIG;
+    }
+
+    /**
+     * Overrides the configuration to be used for the current VList
+     * @param config Config to apply to the VList
+     */
+    setConfiguration(config: VListBehaviorConfig) {
```

Comment:
Can we set this config in constructor instead?

Reply:
Or, only pass preventItemRemovalOnOutdent as a parameter in setIndentation

Reply:
Sounds good, I think only adding the parameter to setIndentation is better